test/sockets.test.js

```diff
@@ -1,63 +1,186 @@
 import 'regenerator-runtime/runtime';
 
 import Socket from '../src/sockets';
-import Logger from '../src/logger';
+import { Logger, SOCKET_PING } from 'syft-helpers.js';
 
 import { WebSocket, Server } from 'mock-socket';
 
 global.WebSocket = WebSocket;
 
 const url = 'ws://localhost:8080/';
 
-const onMessage = event => console.log('NEW MESSAGE', event);
-const onOpen = event => console.log('NEW OPEN', event);
-const onClose = event => console.log('NEW CLOSE', event);
-const onError = event => console.log('NEW ERROR', event);
+/**
```

Comment:
We're not using JSDoc, feel free to throw these comments out for now.

Reply:
OK